README.md

```diff
@@ -1390,6 +1390,12 @@ I`m *[Hanna](https://github.com/HannaSyn)*, and I am really excited to join comm
 ![kitten](https://slm-assets.secondlife.com/assets/16934237/view_large/animals_05.jpg) 
 ---
 
-## What's up Kottans!
-My name is [Alex](https:/github.com/usides) and I want to be a dev-cat.
-![Image of DevCat](https://www.nexmo.com/wp-content/uploads/2016/08/dx-jamie-comic.jpg)
\ No newline at end of file
+## Hi, Kottans!
+I'm Kyrylo, I have always dreamed of programming and this is a great opportunity for me :)
+ ![Cat in the glasses](https://f3.upet.com/o_XSbhGBwd1G_1.jpg)
+ 
+## What's up Kottans! 
+My name is [Alex](https://github.com/usides) and I want to be a dev-cat.
```

Comment:
Still rewriting old changes, please remove indents from 1398 to 1401 at the beginnings of the lines

Reply:
Look, you shouldn't change previously added code
<img width="1514" alt="Screenshot 2020-10-13 at 16 36 10" src="https://user-images.githubusercontent.com/14895118/95868015-75e05100-0d72-11eb-813b-40e4bc453dfe.png">
Now GitHub shows us that you've removed Alex's changes and added them after yours

Reply:
Above they wrote that Alex's code was duplicated and asked me to delete, which I did. Maybe I did something wrong. Maybe I delete this request and submit a new one?
https://github.com/kottans/mock-repo/pull/442#pullrequestreview-507354098
I think that's why the error occurred

Reply:
I'm rewrite this, but on my side I don't see indents. Sorry for that, it's my first pull request 😿

Reply:
It would be great
t.me/Sadpotatoe